public/js/components/FormFields/FormFieldScribeEditor.js

```diff
@@ -27,9 +28,13 @@ export default class FormFieldsScribeEditor extends React.Component {
 
   wordCount = text => text.trim().replace(/<(?:.|\n)*?>/gm, '').split(/\s+/).filter(_ => _.length !== 0).length;
 
+  isTooLong = (wordCount) => this.props.suggestedLength && wordCount > this.props.suggestedLength;
+
   renderWordCount = () => {
-    const wordCount = this.wordCount(this.props.fieldValue);
-    const tooLong = this.props.suggestedLength && wordCount > this.props.suggestedLength;
+    const wordCount = this.props.fieldValue? this.wordCount(this.props.fieldValue) : 0;
+
+    const tooLong = this.isTooLong(wordCount);
```

Comment:
Is this ever used?

Reply:
I believe so, two lines down:

`{tooLong ? <span className="form__message__text--error"> (too long)</span>: false}`

Reply:
oops, I was looking at the diff only, now I can see it in the full source :-)